src/components/LoginForm/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { useForm } from 'react-hook-form';
+import { t } from 'i18next';
+import { Link, Redirect } from 'react-router-dom';
+
+import logo from '../../app/assets/logo.png';
+import ButtonForm from '../ButtonForm';
+import InputText from '../InputText';
+import { emailPattern } from '../../utils/formValidations';
+import { login } from '../../services/userService';
+import useRequest from '../../app/hooks/useRequest';
+import { AUTH_INPUTS } from '../../constants/forms';
+import { PATHS } from '../../constants/paths';
+
+import styles from './styles.module.scss';
+
+function LoginForm() {
+  const [userData, setUserData] = useState();
+  const { register, handleSubmit, errors } = useForm();
+  const [isLoading, submitError, response] = useRequest({ request: login, payload: userData });
+  const onSubmit = handleSubmit(({ email, password }) => {
+    setUserData({
+      email,
+      password
+    });
+  });
+
+  if (response && response.ok && response.headers && response.headers.accessToken) {
+    return <Redirect to="/" />;
+  }
+
+  return (
+    <div className={styles.loginForm}>
+      <img src={logo} className={styles.logo} />
+      {submitError && submitError.length ? (
```

Comment:
You don't need to use a ternary here, you can use normal conditional rendering:
```
{submitError && submitError.length > 0 && (...)
```

The only change you need to make is to explicitly ask for `> 0` so the expression returns a boolean (which is not rendered) instead of the length, which is a number and is rendered (so you would get a 0)